openquake/server/views.py

```diff
@@ -849,31 +864,40 @@ def aristotle_run(request):
     :param request:
         a `django.http.HttpRequest` object containing
         usgs_id, rupture_file,
-        lon, lat, dep, mag, rake, dip, strike, maximum_distance, trt,
+        lon, lat, dep, mag, rake, dip, strike,
+        local_timestamp, time_event,
+        maximum_distance, trt,
         truncation_level, number_of_ground_motion_fields,
-        asset_hazard_distance, ses_seed, station_data_file,
-        maximum_distance_stations
+        asset_hazard_distance, ses_seed,
+        maximum_distance_stations, station_data_file
     """
     res = aristotle_validate(request)
     if isinstance(res, HttpResponse):  # error
         return res
-    (rupdic, maximum_distance, trt,
+    (rupdic, local_timestamp, time_event, maximum_distance, trt,
      truncation_level, number_of_ground_motion_fields,
      asset_hazard_distance, ses_seed, maximum_distance_stations,
      station_data_file) = res
```

Comment:
Having so many positional parameters is not good. We will have to think of something (in another PR).